webapp/scripts/preview/requestBody.js

```diff
@@ -193,7 +193,12 @@ HeadersTab.prototype = domplate(TabView.Tab.prototype,
         var url = this.file.request.url;
         if (url) {
             var anchor = $(body).find("a")[0];
-            anchor.setAttribute("href", url);
+            if (url.includes("<REDACTED>")) {
+              anchor.setAttribute("href", "");
+              anchor.setAttribute("class", anchor.getAttribute("class") + " disableLink");
+            } else {
+              anchor.setAttribute("href", url);
+            }
```

Comment:
We opted to maintain this as an `a` tag just in case something else in the harviewer, or app code, relies on this being a link. Since we are looking for `<REDACTED>` in the URL, this seems safe in regards to backwards compatibility, and it will automatically fix optimization.